uspace/drv/block/ahci/ahci.c

```diff
@@ -108,22 +108,22 @@
 		.cmd = CMD_ACCEPT \
 	}
 
-static int get_sata_device_name(ddf_fun_t *, size_t, char *);
-static int get_num_blocks(ddf_fun_t *, uint64_t *);
-static int get_block_size(ddf_fun_t *, size_t *);
-static int read_blocks(ddf_fun_t *, uint64_t, size_t, void *);
-static int write_blocks(ddf_fun_t *, uint64_t, size_t, void *);
+static errno_t get_sata_device_name(ddf_fun_t *, size_t, char *);
+static errno_t get_num_blocks(ddf_fun_t *, uint64_t *);
+static errno_t get_block_size(ddf_fun_t *, size_t *);
+static errno_t read_blocks(ddf_fun_t *, uint64_t, size_t, void *);
+static errno_t write_blocks(ddf_fun_t *, uint64_t, size_t, void *);
 
-static int ahci_identify_device(sata_dev_t *);
-static int ahci_set_highest_ultra_dma_mode(sata_dev_t *);
-static int ahci_rb_fpdma(sata_dev_t *, uintptr_t, uint64_t);
-static int ahci_wb_fpdma(sata_dev_t *, uintptr_t, uint64_t);
+static errno_t ahci_identify_device(sata_dev_t *);
+static errno_t ahci_set_highest_ultra_dma_mode(sata_dev_t *);
+static errno_t ahci_rb_fpdma(sata_dev_t *, uintptr_t, uint64_t);
+static errno_t ahci_wb_fpdma(sata_dev_t *, uintptr_t, uint64_t);
 
 static void ahci_sata_devices_create(ahci_dev_t *, ddf_dev_t *);
 static ahci_dev_t *ahci_ahci_create(ddf_dev_t *);
 static void ahci_ahci_hw_start(ahci_dev_t *);
 
-static int ahci_dev_add(ddf_dev_t *);
+static errno_t ahci_dev_add(ddf_dev_t *);
```

Comment:
I think there is a newly introduced trailing ws here

Reply:
It's not new.